### PR TITLE
[CPU offload] correct cpu offload

### PR DESCRIPTION
**Pull request by patrickvonplaten**
Correct a bug related to CPU offload as discovered by @Sanster here: https://github.com/huggingface/diffusers/issues/1934

### CODE REVIEW

**Comment by patrickvonplaten** [comment]
Also cc @piEsposito just FYI :-)

File: src/diffusers/pipelines/alt_diffusion/pipeline_alt_diffusion.py [hunk 1]
```diff
@@ -211,13 +211,10 @@ def enable_sequential_cpu_offload(self, gpu_id=0):
...
         for cpu_offloaded_model in [self.unet, self.text_encoder, self.vae]:
-            if cpu_offloaded_model is not None:
-                cpu_offload(cpu_offloaded_model, device)
+            cpu_offload(cpu_offloaded_model, device)
 
         if self.safety_checker is not None:
-            # TODO(Patrick) - there is currently a bug with cpu offload of nn.Parameter in accelerate
-            # fix by only offloading self.safety_checker for now
-            cpu_offload(self.safety_checker.vision_model, device)
+            cpu_offload(self.safety_checker, execution_device=device, offload_buffers=True)
```
Comment (pcuenca) [inline src/diffusers/pipelines/alt_diffusion/pipeline_alt_diffusion.py:217]:
```suggestion
            cpu_offload(self.safety_checker, device, offload_buffers=True)
```

nit: just making it match the previous invocation for clarity (`device` is `execution_device`).

**Review by pcuenca — APPROVED** [review]
Very cool!

**Comment by patil-suraj** [comment]
Thanks for fixing !

**Pull request merged**